tests/integration/dynamic_ip_test.py

```diff
@@ -134,16 +131,22 @@ def dhcpcli_up(dhcp_env):
         yield ifstate
 
 
+@pytest.fixture
+def dhcp_sb_cli_up(dhcp_env_vlan):
```

Comment:
What does the `sb` here mean?

Reply:
southbound.

would you propose a better name?

Reply:
southbound of what? This fixture/func does not know how the return value is used, so naming it relative to something that may use it is problematic.

How about `dhcp_client_iface_up`.